specification/servicefabricmanagedclusters/resource-manager/readme.md

```diff
@@ -201,6 +212,17 @@ input-file:
 - Microsoft.ServiceFabric/preview/2021-01-01-preview/nodetype.json
 ```
 
+### AutoRest v3 Suppressions
+``` yaml
+suppressions:
+    
+  - code: ResourceNameRestriction
+    reason: the service does not have a patter restriction. ARM's default rule should get applied.
+
+  - code: PutResponseSchemaDescription
+    reason: service creted with the 202 patter before this rule was introduced.
+```
+
 ---
```

Comment:
I'm confused about what exactly you're suppressing here.

Reply:
on put request that are long running operations we were getting flagged because we don't return 201. The model we have is described below (which we validated with ARM when we initially created it):

- For new cluster, return 200 with ProvisioningState = Creating
 - For existing cluster, return 202 Accepted + Location header, which links to a ManagedClusterOperationResult resource that:
     - returns 202 if the update operation is still running
     - returns 200 if the update operation is completed (resource transition to terminal state)
     
 @rkmanda, suggested to add add this suppression "If your service is old and you are carrying forward the 202 pattern, add a suppression and we will approve it."